src/internal/event/EventCreators.ts

```diff
@@ -74,13 +74,13 @@ export const newGoalEvent = (
 
 export const newLatencyMetricsData = (
   apiId: ApiId,
-  duration: number,
+  latencySecond: number,
   featureTag: string,
 ): LatencyMetricsEvent => {
   return {
     apiId,
     labels: { tag: featureTag },
-    duration: { value: duration },
+    latencySecond: { value: latencySecond },
```

Comment:
Sorry, I misunderstood the duration type.
The `latencySecond` is not an object. It is a number type.
Can you change its type?

Reply:
Sure, fixed: [`9092541` (#41)](https://github.com/bucketeer-io/javascript-client-sdk/pull/41/commits/9092541512757ffd05d5716682ed14022148c336)